pkg/odo/cli/catalog/list/components.go

```diff
@@ -42,7 +42,7 @@ func NewListComponentsOptions() *ListComponentsOptions {
 func (o *ListComponentsOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
 	tasks := util.NewConcurrentTasks(2)
 
-	if util.IsValidKubeConfigPath() {
+	if err1 := util.IsValidKubeConfigPath(); err1 == nil {
```

Comment:
@dharmit  The only reason I am using different variable names is to avoid the duplication warning from sonar cloud. I dont mind changing it back if we are going to ignore those warnings

Reply:
@valaparthvi there's something that needs to be fixed in SonarCloud here. I don't see what @mohammedzee1000 is doing wrong in using the variable name as `err` across a single function.

@feloy do you have any clues if I'm interpreting this incorrectly?

Reply:
I think we should try to fix the Duplication problem instead. The idiomatic way in Go is to reuse the `err` variable and not create new ones for every instruction.

Reply:
And that is how it was. The reason I did this change as it was complaining about 
`err=util.IsValidKubeConfigPath()` across multiple files.

It also complains about imports

Reply:
That's weird. Sonar should not complaint about imports being duplicated. In any case, if the duplication errors thrown by Sonar seem irrelevant, like in your case, please don't feel compel to fix it. Until we can figure out a way to configure Sonar to throw errors in the idiomatic Go way, I would suggest only fix the relevant errors.

Reply:
But, I do appreciate that you are taking the suggestions by Sonar into consideration and working on fixing them :)